src/components/UI/AccountAddress/AccountAddress.js

```diff
@@ -23,7 +23,14 @@ export const AccountAddress = ({address, onClick}) => {
   return (
     <>
       <div ref={ref} className={styles.accountAddress}>
-        {address.length <= 42 ? address : `${address.substring(0, 42)}...`}
+        <div className={styles.address}>
+          {address.length < 40
+            ? address
+            : `${address.substring(0, 24)}...${address.substring(
+                address.length - 12,
+                address.length
```

Comment:
Use `shortenAddress` from utils?

Reply:
We can generalize the function.
`export const shortenAddress = (account: string, maxDigits: number, startDigits: number): string`
Or something like that.

Reply:
done